AudioKit/Common/Nodes/Playback/Player/ClipPlayer/AKClip.swift

```diff
@@ -61,23 +62,29 @@ open class AKFileClip: NSObject, FileClip {
     /// The duration of playback.
     open var duration: Double
 
+    // The callback executed when the end of the clip is reached.
+    open var completionCallback: AKCallback?
```

Comment:
I think it should be called completion rather than completionCallback, this name along with the type makes it clear what it's purpose is.